index.html

```diff
@@ -14,18 +14,12 @@
 
     Hello Camper!
 
-    For now, the test suite only works in Chrome!
-    Please read the README below in the JS Editor before beginning.
-    Feel free to delete this message once you have read it.
-    Good luck and Happy Coding!
-
-    - The freeCodeCamp Team
-
+    Microverse team
     -->
 
 
     <header>
-      <h1 id="title">Novel And Nadir Survey Form</h1>
+      <h1 id="title">Novel And Nadir Survey Form for own review</h1>
```

Comment:
This is an example inline comment. 🙂 

You can see how it's under the main summary above and point to a specific part of the code where you want to suggest changes or just comment on it generally. 😉 

To do this, make sure you go to the "File changes" tab and use the "Start a review" button for your first review comment.